about.html

```diff
@@ -0,0 +1,237 @@
+                            </ul>
+
+                        </div>
+                        <button type="button" class=" campaign-button d-none d-md-block">
+                            Plan</button>
+                    </div>
+                </div>
+            </section>
+        </nav>
+        <div class="behind-navbar"></div>
+
+        <div class="opacity">
+
+            <button type="button" class="ham">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+                </svg>
+            </button>
+        </div>
+
+        </div>
+
+        <div class="overlayy"></div>
+        <nav class="sideNav">
+            <button type="button" class="menuX">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                </svg>
+            </button>
+            <ul>
+                <li><a class="menuLink" href="index.html">Home</a></li>
+                <li><a class="menuLink" href="about.html">About</a></li>
+                <li><a class="menuLink" href="#projects-container">Contact</a></li>
+                <li><a class="menuLink" href="#about-container">Sponsort</a></li>
+                <li><a class="menuLink main-button" href="#footer">Buy Tickets</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="hero-section background-image ">
+            <div class="opacity">
+                <div class="hero-contenedor-info container-fluid">
+                    <h4 class="text-primary-color text-center mb-5">&#34;The Chance Concert&#34;</h4>
+                    <h1 class="titles-font-family text-primary-color text-center">LIVE MUSIC AND Concert 2022 </h1>
+                    <div class="info-hero">
+                        <p class="text-secondary-color"> The chance showcases popular and established musical artists as
+                            well as emerging artists and reunited groups. It is one of the largest, most famous, and
+                            most profitable music festivals around the world</p>
+                    </div>
+                    <div class="bottom-info-hero">
+                        <p>Please contact us per email for any further questions about festival
+                        </p>
+                        <a href="mailto:info@loachella.fest" class="d-flex align-items-center justify-content-center">
+                            <p class="email font-weight-bold mr-2">info@loachella.fest</p> <i
+                                class="fas fa-angle-right"></i>
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- _(optional)_Hi. Great job. I'll recommend changing the colors because, from an accessibility point of view, a red text on a red background Is not very easy to read 
- **imp**
- 
![image](https://user-images.githubusercontent.com/37901469/169626466-9e602920-37b2-4dc6-92bf-1e8fe7a39346.png)

Reply:
ok thanks this was my last review